stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1361,4 +1361,61 @@
     (begin
         (ok (map-set aggregate-public-keys reward-cycle aggregate-public-key))
     )
-)
\ No newline at end of file
+)
+
+;;
+;; @contract voting for the aggregated public key
+;;
+
+;; maps dkg round and signer to proposed aggregated public key
+(define-map votes {reward-cycle: uint, round: uint, signer: principal} {aggregated-public-key: (buff 33), reward-slots: uint})
+;; maps dkg rount and aggregated public key to weights of signers supporting this key so far
+(define-map tally {reward-cycle: uint, round: uint, aggregate-public-key: (buff 33)} uint)
+
+(define-constant err-not-allowed (err u10000))
```

Comment:
Can you relocate to batch w/ rest of errors & update the format to match existing? (capitalized, currently in the 30s index).

`(define-constant ERR_INCORRECT_REWARD_CYCLE 32)`